### PR TITLE
Sciome update: sparseNN bug fix and new tests

### PR DESCRIPTION
**Pull request by ericbair-sciome**

### CODE REVIEW

File: tests/testthat/test-Log_Likelihood.R
Comment (Spatiotemporal-Exposures-and-Toxicology) [inline tests/testthat/test-Log_Likelihood.R:1]: tests look good on their face. I'm going to add the codecov.io coverage reports to the yaml. This should start providing line by line coverage reports to the repo. You can also get this coverage locally via covr package. Have you done that?

File: tests/testthat/test-PrestoGP_Spatial_Model.R
Comment (Spatiotemporal-Exposures-and-Toxicology) [inline tests/testthat/test-PrestoGP_Spatial_Model.R:1]: @ericbair-sciome @shail-choksi @sciome-bot I don't see calls to any expect_* function in any of these tests. Is the tryCatch error call a different kind of test?

File: tests/testthat/test-PrestoGP_Spatiotemporal_Model.R
Comment (Spatiotemporal-Exposures-and-Toxicology) [inline tests/testthat/test-PrestoGP_Spatiotemporal_Model.R:1]: @ericbair-sciome what is the point of "return(1)" in many of these?

Comment (ericbair-sciome) [reply tests/testthat/test-PrestoGP_Spatiotemporal_Model.R:1]: That tells R to skip those tests for the time being. I didn't want the testing to return a bunch of errors for functions that we haven't worked on yet. Those will be deleted in the future once the functions (and corresponding tests) are working correctly.

File: tests/testthat/test-PrestoGP_Spatial_Model.R
Comment (ericbair-sciome) [reply tests/testthat/test-PrestoGP_Spatial_Model.R:1]: There is an expect_error statement later on. By default it only shows the lines that have changed, so the expect_error line isn't visible unless you click on it. (And, as noted below, those tests are being skipped for the time being anyway.)

File: tests/testthat/test-Log_Likelihood.R
Comment (ericbair-sciome) [reply tests/testthat/test-Log_Likelihood.R:1]: I have not done that. That is a good suggestion.

File: tests/testthat/test-PrestoGP_Spatial_Model.R
Comment (Spatiotemporal-Exposures-and-Toxicology) [reply tests/testthat/test-PrestoGP_Spatial_Model.R:1]: ah, sorry about that. Thank you!

File: tests/testthat/test-Log_Likelihood.R
Comment (Spatiotemporal-Exposures-and-Toxicology) [reply tests/testthat/test-Log_Likelihood.R:1]: Cool - As I mentioned, I'll get that going in due time.

**Pull request merged**